R/AddCardModule.R

```diff
@@ -89,6 +89,13 @@ add_card_button_srv <- function(id, reporter, card_fun) {
         shiny::modalDialog(
           easyClose = TRUE,
           shiny::tags$h3("Add a Card to the Report"),
+          shiny::textAreaInput(
```

Comment:
Simple Text Input is enough it will be a one Phrase we have to think what to do with the long ones.